packages/renderer/src/lib/node/node-utils.ts

```diff
@@ -60,6 +60,7 @@ export class NodeUtils {
     const containerRuntime = node.status?.nodeInfo?.containerRuntimeVersion ?? '';
 
     return {
+      kind: 'Node',
       uid: node.metadata?.uid ?? '',
```

Comment:
does we properly have on the renderer side an `uid` for each object?

I feel like we really should have the uid object, this is something important in Kubernetes

> Each [object](https://kubernetes.io/docs/concepts/overview/working-with-objects/#kubernetes-objects) in your cluster has a [Name](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names) that is unique for that type of resource. Every Kubernetes object also has a [UID](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#uids) that is unique across your whole cluster.

If we do not have each Info/UI object with it, we should probably have an issue to add them all somewhere

https://kubernetes.io/docs/concepts/overview/working-with-objects/names/

Reply:
I think I only saw uuid on maybe 2-3 of the current 10 objects. Since we are using it on those (like line 64 above) I'd expect it is there correctly for all the others though.

I agree we should add it, just wasn't the primary focus here / isn't needed for any specific use that I'm aware of atm, so I'd prefer to leave that for future PR.

Reply:
Yeah future PR is okay for me

Reply:
I vote for having the `uid` in the UI version of the object.

https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#uids

It will be present for all objects, and is important if we want to distinguish different versions of a resource with a same name (a user can delete a resource and create it again with the same name, in this case only the uid is helpful to distinguish them). It is also used to associate Events to resources, or to reference owner via `ownerReferences`, and probably more